iOS/rainbow/View/Base.lproj/Main.storyboard

```diff
@@ -321,136 +321,163 @@
-                                <rect key="frame" x="16" y="304" width="119" height="34"/>
-                                <constraints>
-                                    <constraint firstAttribute="width" constant="119" id="PVu-jv-o3F"/>
-                                    <constraint firstAttribute="height" constant="34" id="Y8O-qz-LMU"/>
-                                </constraints>
-                                <fontDescription key="fontDescription" name="IBMPlexSans" family="IBM Plex Sans" pointSize="17"/>
-                                <nil key="textColor"/>
-                                <nil key="highlightedColor"/>
-                            </label>
-                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="0%" textAlignment="center" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="UEf-HW-pqO">
-                                <rect key="frame" x="143" y="304" width="216" height="34"/>
-                                <constraints>
-                                    <constraint firstAttribute="height" constant="34" id="hUY-dM-FUI"/>
-                                </constraints>
-                                <fontDescription key="fontDescription" name="IBMPlexSans-Bold" family="IBM Plex Sans" pointSize="20"/>
-                                <nil key="textColor"/>
-                                <nil key="highlightedColor"/>
-                            </label>
-                            <slider opaque="NO" contentMode="scaleToFill" enabled="NO" contentHorizontalAlignment="center" contentVerticalAlignment="center" minValue="0.0" maxValue="1" translatesAutoresizingMaskIntoConstraints="NO" id="64Y-c9-9yw">
-                                <rect key="frame" x="14" y="374" width="347" height="30"/>
-                                <constraints>
-                                    <constraint firstAttribute="height" constant="29" id="iHD-VQ-97a"/>
-                                </constraints>
-                            </slider>
-                            <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="X0Q-em-HSV">
-                                <rect key="frame" x="16" y="543" width="343" height="42"/>
-                                <color key="backgroundColor" red="0.0" green="0.61176470589999998" blue="0.87450980389999999" alpha="1" colorSpace="calibratedRGB"/>
+                            <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" translatesAutoresizingMaskIntoConstraints="NO" id="j0j-Xz-mVk">
```

Comment:
I've tried this build and there is an auto-layout issue with some conflicting constraints. Please make the other changes and I'll take care of the auto-layout issues afterwards on the same branch.